src/mlpack/methods/bayesian_ridge/bayesian_ridge.cpp

```diff
@@ -0,0 +1,297 @@
+/**
+ * @file bayesian_ridge.cpp
+ * @author Clement Mercier 
+ *
+ * Implementation of Bayesian Ridge regression.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include "bayesian_ridge.hpp"
+#include <mlpack/core/util/log.hpp>
+#include <mlpack/core/util/timers.hpp>
+
+using namespace mlpack;
+using namespace mlpack::regression;
+
+
+BayesianRidge::BayesianRidge(const bool fitIntercept,
+			     const bool normalize) :
+  fitIntercept(fitIntercept),
+  normalize(normalize)
+{
+  Log::Info << "Baysian Ridge regression(fitIntercept="
+	    << this->fitIntercept
```

Comment:
Idendation should be 4 spaces (tabbed twice); see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more details. Also we don't print class information inside the constructor itself, I guess a good place would be the executable.